outline/setup_osx.md

```diff
@@ -86,32 +83,17 @@ Now run `which lein` and you should see the `lein` command.
 
 After you set up Leiningen as above, run the `lein version` command. This should take a while to run, as it will download some resources it needs the first time. If it completes successfully, you are golden! If not, ask an instructor for help.
 
-## Installing Light Table
+## Installing Nightcode
 
-Go to the [LightTable site](http://www.lighttable.com/). On the page there, you should see a set of buttons that have download links for Light Table. Click the "OS X 10.7+" button and you will download a .zip file.
+Go to the [Nightcode site](https://sekao.net/nightcode/). On the page there, click the "Free Download" link and you will download a .jar file. There should now be a file named `nightcode-0.4.7-standalone.jar` in your Downloads folder. If Java is installed, you can double click the JAR file to run it. Depending on your Mac's security settings, you may see an error message that says the author of the file can't be verified. You'll be able to run Nightcode if you relax the security settings.
```

Comment:
(Some/most/all) OSX users will have the option of getting Nightcode from the App store. Their setup should be simpler, but we'll need to repeat this good work for that install flow.